Formula/zydis.rb

```diff
@@ -0,0 +1,24 @@
+class Zydis < Formula
+  desc "Fast and lightweight x86/x86-64 disassembler library"
+  homepage "https://zydis.re"
+  url "https://github.com/zyantific/zydis/archive/v2.0.2.tar.gz"
+  sha256 "bd711102a5a30096562a7cb60bafbc9c4a2441ce5463a59f4d16f2dd73f9fb72"
+  head "https://github.com/zyantific/zydis.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "zydis-build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+      bin.install "ZydisDisasm"
+      bin.install "ZydisInfo"
+      bin.install "ZydisFuzzIn"
+      bin.install "ZydisPerfTest"
+    end
+  end
+
+  test do
+    system "#{bin}/ZydisInfo"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
improved the test to compare with expected output

Reply:
It's now a great test